package.json

```diff
@@ -4,7 +4,8 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "eject": "expo eject"
+    "lint": "eslint **/*.js",
+    "eject": "expo eject do not eject"
```

Comment:
We can just remove this to prevent people from ejecting.

Reply:
i will update the PR with the changes, thanks for the pointers CJ